lib/inspector.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
-const connect = process.binding('inspector').connect;
 const EventEmitter = require('events');
+const connect = process.binding('inspector').connect;
```

Comment:
I tend to write code to get clean diffs and minimize merge conflicts by not putting multiple definitions on the same line, but if the above is the preferred style, np, I can do.